src/together/cli/api/finetune.py

```diff
@@ -60,12 +60,30 @@ def fine_tuning(ctx: click.Context) -> None:
 )
 @click.option("--batch-size", type=INT_WITH_MAX, default="max", help="Train batch size")
 @click.option("--learning-rate", type=float, default=1e-5, help="Learning rate")
+@click.option(
+    "--min-lr-ratio",
+    type=float,
+    default=0.0,
+    help="Final learning rate as a percentage of the initial learning rate",
```

Comment:
Is it ratio (0.05 means 5%) or percentage? Or value for min learning rate? Min lr would be more concise in my opinion, but we could specify helpstring somewhat differently I believe

Reply:
I was also confused about exposing the ratio instead of the absolute LR — this seems to be how optimizer schedules are frequently configured (even in Axolotl, there is `cosine_min_lr_ratio` as a config option). However, I agree that "percentage" is misleading, because this ratio goes from 0 to 1

Reply:
Good point, removed "percentage" here and will update in the together-finetune PR as well